webapp/src/components/content/checkboxElement.tsx

```diff
@@ -86,6 +86,7 @@ const CheckboxElement = (props: Props) => {
                 }}
                 readonly={readonly}
                 spellCheck={true}
+                className=''
```

Comment:
alternatively, could check for an `undefined` `className` prop in the `Editable` component itself to avoid needing to pass in am empty string literal (similar to how `readonly` is handled https://github.com/mattermost/focalboard/blob/923437cc573ca515e17c67b56b09436ae9014e0c/webapp/src/widgets/editable.tsx#L90)

the primary benefit being we could wholly omit the `className` prop if it's not needed for a component

Reply:
Yes, this can be the alternate and best solution. Thank you @Pinjasaur, I will make the changes and push it as you suggested.